docs/multicluster/antctl.md

```diff
@@ -0,0 +1,26 @@
+# Antctl Multi-cluster commands
+
+Starting from version 1.6.0, Antrea supports the `antctl mc` commands, which can collect information from a leader cluster in a ClusterSet, for troubleshooting issues in an Antrea Multi-cluster ClusterSet.
```

Comment:
Could you wrap the lines like other docs? Typically for docs we limit 80 chars per line so we could see the whole lines in some editors. But you don't need to break lines for code block and link.

Reply:
done